server/cmwell-ws/app/security/AuthCache.scala

```diff
@@ -26,23 +26,40 @@ import play.api.libs.json.{JsValue, Json}
 
 import scala.concurrent.duration._
 import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.util.Try
 
 @Singleton
 class AuthCache @Inject()(crudServiceFS: CRUDServiceFS)(implicit ec: ExecutionContext) extends LazyLogging {
   // TODO Do not Await.result... These should return Future[Option[JsValue]]]
-  def getRole(roleName: String, nbg: Boolean): Option[JsValue] = Await.result(data(nbg).getAndUpdateIfNeeded.map(_.roles.get(roleName)).recoverWith { case _ =>
-    logger.warn(s"AuthCache Graceful Degradation: Search failed! Trying direct read for Role($roleName):")
-    getFromCrudAndExtractJson(s"/meta/auth/roles/$roleName", nbg)
-  }, 5.seconds)
+  def getRole(roleName: String, nbg: Boolean): Option[JsValue] = {
+    val esDependingFut = data(nbg).getAndUpdateIfNeeded.map(_.roles.get(roleName))
+    Await.result(
```

Comment:
Are we still using Await?
can we format it as:
```
val f = ...
Await.result(f,6.seconds)
```
instead of:
```
Await.result(
  ...
  , 6.seconds
)
```

Reply:
Yes, I kept that `TODO` to some other time. And yes, I can do that `val f`.